src/solver_enums.cpp

```diff
@@ -51,6 +51,17 @@ const unordered_map<SolverEnum, unordered_set<SolverAttribute>>
               QUANTIFIERS
           } },
 
+        { GENERIC_SOLVER,
+          {
+              TERMITER,
+              THEORY_INT,
```

Comment:
So what did we decide about this again? What if the generic solver is instantiated with a solver that doesn't support integers? Is this actually just the CVC4 supported attributes?

Reply:
These are the attributes that the generic *interface* supports. It can handle, e.g., integers, but it cannot handle term transfer. So this is a property of the generic interface, not of any particular instantiation of it (which would be impossible to predict). Does that make sense?

Reply:
I see, makes sense!